core/sass/components/_header.scss

```diff
@@ -0,0 +1,175 @@
+// Main Site Header 
```

Comment:
We are using https://github.com/uktrade/directory-header-footer for header/footer on FAB/FAS/Exred/SSO/SOO - @eadpearce can advise on that.

If this is for the international page, the naming of the files and descriptions need to reflect that.

Reply:
The site is supposed to be using components/header and components/footer directly within the same repo. However, since directory-ui-export-readiness hasn't been built properly using the structure and files supplied from exred repo, I've now simply incorporated the specific CSS from those shared _header.scss and _footer.scss files directly into the international.scss